vmware-base/src/main/java/com/cloud/hypervisor/vmware/mo/ScsiDiskControllerType.java

```diff
@@ -17,8 +17,10 @@
 package com.cloud.hypervisor.vmware.mo;
 
 public interface ScsiDiskControllerType {
-    public final static String LSILOGIC_PARALLEL = "lsilogic";
-    public final static String LSILOGIC_SAS = "lsisas1068";
-    public final static String BUSLOGIC = "buslogic";
-    public final static String VMWARE_PARAVIRTUAL = "pvscsi";
+    String LSILOGIC_PARALLEL = "lsilogic";
+    String LSILOGIC_SAS = "lsilogicsas";
+    String LSILOGIC_SAS_1068 = "lsisas1068";
+    String BUSLOGIC = "buslogic";
+    String VMWARE_PARAVIRTUAL = "pvscsi";
+    String VMWARE_VIRTUAL_SCSI = "VirtualSCSI";
```

Comment:
@nvazquez is it possible to keep this as  'scsi' simply (in line with disk controller type) ?

Reply:
Renamed